vlib/v/checker/tests/generic_fn_generic_name_unresolved_err.out

```diff
@@ -0,0 +1,21 @@
+vlib/v/checker/tests/generic_fn_generic_name_unresolved_err.vv:8:7: error: generic fn using generic types cannot be called outside of generic fn
+    6 | 
+    7 | fn main() {
+    8 |     _ := func[T]
```

Comment:
Just `func[T]` is not calling the function, it is just using it. I think the error message should be adjusted.

Reply:
Yes, I simply copied the older message